doc.go

```diff
@@ -0,0 +1,16 @@
+// Copyright 2016 The prometheus-operator Authors
```

Comment:
Nit: This is a new file so copyright starts from merge date:

```suggestion
// Copyright 2019 The prometheus-operator Authors
```